doc/api/process.md

```diff
@@ -530,16 +530,14 @@ running the `./configure` script.
 An example of the possible output looks like:
 
 ```js
-{
-  target_defaults:
+{ target_defaults:
```

Comment:
This kind of style shift is actually *against* what the PR stands for, since the diff would be inflated if the first property was removed. (That's not to say I'm against this new style -- In fact I prefer it over the old one -- but consistency is key in documentation.)